ovos_persona_server/persona.py

```
@@ -0,0 +1,36 @@
+    not found, or invalid.
+
+    Returns:
+        Persona: The loaded OVOS Persona instance.
+
+    Raises:
+        HTTPException: If persona configuration or loading fails.
+    """
+    global default_persona
+    if default_persona is None:
+        raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+                            detail=f"Failed to load persona")
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)